apps/src/applab/applab.js

```diff
@@ -367,12 +367,24 @@ function shouldRenderFooter() {
   return studioApp.share;
 }
 
+const APPLAB_URL_PATTERN = /^(.*\/projects\/applab\/[A-Za-z0-9-]+)\/.*/;
```

Comment:
I'd also go ahead and make this generic enough to work for other types of projects too: `...\/projects\/\w+\/[...`